cmd/internal/cli/startvm.go

```diff
@@ -85,6 +85,7 @@ func startVM(sifImage, singAction, cliExtra string, isInternal bool) error {
 			//Program exited with non-zero return code
 			if status, ok := exitErr.Sys().(syscall.WaitStatus); ok {
 				sylog.Debugf("Process exited with non-zero return code: %d\n", status.ExitStatus())
+                return nil
```

Comment:
Gofmt will complain about this...

Fix: 
```
$ gofmt -w cmd/internal/cli/startvm.go
```

Reply:
Gah...  It *looked* right in the editor

Reply:
I know!!! gofmt like tabs :man_facepalming: